sonar/modules/documents/jsonschemas/documents/document-v1.0.0_src.json

```diff
@@ -1045,12 +1221,61 @@
                   "title": "Contribution",
                   "type": "string",
                   "minLength": 1
+                },
+                "form": {
+                  "hide": true
                 }
               }
-            }
+            },
+            "propertiesOrder": [
+              "title",
+              "publication",
+              "contribution"
+            ],
+            "required": [
+              "title"
+            ]
           }
+        },
+        "propertiesOrder": [
+          "document",
+          "numberingYear",
+          "numberingPages",
+          "numberingVolume",
+          "numberingIssue"
+        ]
+      },
+      "form": {
+        "hide": true,
+        "navigation": {
+          "essential": true
         }
       }
     }
-  }
+  },
+  "propertiesOrder": [
+    "documentType",
+    "title",
+    "organisation",
+    "classification",
+    "language",
+    "abstracts",
+    "subjects",
+    "identifiedBy",
+    "contribution",
+    "dissertation",
+    "editionStatement",
+    "otherEdition",
+    "provisionActivity",
+    "partOf",
+    "specificCollections",
+    "extent",
+    "formats",
+    "notes",
+    "series",
+    "additionalMaterials",
+    "contentNote",
+    "otherMaterialCharacteristics",
+    "usageAndAccessPolicy"
+  ]
```

Comment:
maybe better to place `propertiesOrder` tag just next to required field ?

Reply:
I chose this order: 

1. properties
2. propertiesOrder
3. required

The same should be applied for all the properties. What is you opinion about that?

Reply:
It's not a problem for me if all project schemas follow the same pattern. 
I think it's more easy to find "required" and "propertiesOrder" at the top of the file (easier to find). That's my personal opinion, not a guideline :-)

Reply:
Accordingly to the freshly approved section https://github.com/rero/developer-resources/blob/master/guidelines/json-schemas.md#order, I did the changes.